src/components/Display.js

```diff
@@ -5,6 +5,14 @@ import DisplayPageInfo from './DisplayPages/DisplayPageInfo'
 import DisplayPageMain from './DisplayPages/DisplayPageMain'
 import DisplayPagePad from './DisplayPages/DisplayPagePad'
 
+import PropTypes from 'prop-types'
+
+Display.propTypes = {
+  selectedPad: PropTypes.string,
+  masterVolume: PropTypes.number.isRequired,
```

Comment:
I get a warning in the console: Failed prop type: Invalid prop `masterVolume` of type `string` supplied to `Display`, expected `number`. 
That's weird as you declared it as a number.